packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1784,7 +1799,7 @@ function flushRoot(root: FiberRoot, expirationTime: ExpirationTime) {
   performWorkOnRoot(root, expirationTime, false);
   // Flush any sync work that was scheduled by lifecycles
   performSyncWork();
-  finishRendering();
```

Comment:
This change wasn't strictly necessary, but I noticed while tracing through that this `finishRendering` call was redundant, since `performWork` (called by `performSyncWork`) also calls `finishRendering`.